src/overlays/actors/ovl_Obj_Makeoshihiki/z_obj_makeoshihiki.c

```diff
@@ -78,9 +79,9 @@ void ObjMakeoshihiki_Init(Actor* thisx, PlayState* play) {
 
     if (Actor_SpawnAsChild(&play->actorCtx, thisx, play, ACTOR_OBJ_OSHIHIKI, spawnPos->x, spawnPos->y, spawnPos->z, 0,
                            block->rotY, 0, ((block->color << 6) & 0xC0) | (block->type & 0xF) | 0xFF00) == NULL) {
-        // "Push-pull block failure"
         PRINTF_COLOR_ERROR();
-        PRINTF("Ｅｒｒｏｒ : 押し引きブロック発生失敗(%s %d)\n", "../z_obj_makeoshihiki.c", 194);
+        PRINTF(T("Ｅｒｒｏｒ : 押し引きブロック発生失敗(%s %d)\n", "Error : Push/pull block failure (%s %d)\n"),
+               "../z_obj_makeoshihiki.c", 194);
```

Comment:
Earlier this was written as push-pull. I dont have a preference either way, just do think it should be consistent.
Maybe i lean slightly toward using a slash

Reply:
seems there are a bunch more in other files too